Bottom: 04f9d58c1850f18484c723d68bfe7370ff70f7de
Top:    09bd997137a4cd666e1e9b9cf767ac9fcf1d3169
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-11 22:38:29 +0000

Refresh of toy-rendering-attempt

---

diff --git a/src/hid/step/model.cpp b/src/hid/step/model.cpp
index efa3a11..52aa455 100644
--- a/src/hid/step/model.cpp
+++ b/src/hid/step/model.cpp
@@ -1054,8 +1054,8 @@ process_sr_or_subtype(InstMgr *instance_list, SdaiShape_representation *sr, proc
       ry = ((RealNode *)parent_axis->ref_direction_ ()->direction_ratios_ ()->GetHead ()->NextNode ())->value;
       rz = ((RealNode *)parent_axis->ref_direction_ ()->direction_ratios_ ()->GetHead ()->NextNode ()->NextNode ())->value;
 
-      rotate_basis_inverted (info->current_transform, ax, ay, az, rx, ry, rz);
       translate_origin (info->current_transform, -ox, -oy, -oz);
+      rotate_basis_inverted (info->current_transform, ax, ay, az, rx, ry, rz);
 
       /* XXX: Actually only want to "process" the SR once per SR, then create _instances_ of it */
       /* XXX: Origin offset etc..? */
